test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -341,6 +345,8 @@ def test_res_size(self):
         Test that server_dyn_res accepts type "size" and a "value"
         returned by a script
         """
+        a = {'resources_available.ncpus': 2}
+        self.server.manager(MGR_CMD_SET, NODE, a, id=self.mom.shortname)
```

Comment:
if we want one job to run and for other we want "Can never run", we do not need the first job to be in 'R' state when we submit the second job. We can do away with this as well.

Reply:
@prakashcv13 In the test server dynamic resource script has 100gb as the value for resource foobar and in the end we are submitting 2 jobs requesting foobar=50gb and verifying both the job should run. That's why I have added code for setting ncpus

Reply:
ok - I did not see the last 2 jobs.
I am of the view that we do not need that to be tested as from the first two jobs it is clear that "server_dyn_res accepts type "size" and a "value" returned by a script" which is the purpose of this test case.
What do you think?

Reply:
@prakashcv13 Yeah the initial job submissions would verify the intent of the test but I am thinking that the last 2 jobs submitted with a intention to verify that jobs can run when resource(foobar) requested is 100gb in total(which is the limit set)

Reply:
in that case submit one job that uses 100gb. unless the test case itself is that more than one job must run, we can do that.